bucket/prismlauncher-qt5.json

```diff
@@ -26,10 +41,24 @@
             "Prism Launcher"
         ]
     ],
+    "persist": [
```

Comment:
Prism Launcher 7.1 now has a `logs` folder, this should be added to `persist`.

Reply:
done!